src/Common/tests/System/Net/Configuration.Certificates.cs

```diff
@@ -18,6 +18,8 @@ public static partial class Certificates
         {
             private const string CertificatePassword = "testcertificate";
             private const string TestDataFolder = "TestData";
+            private static Mutex m = new Mutex(false, "Global\\CoreFXTest.Configuration.Certificates.LoadPfxCertificate");
+            private const int MutexTimeout = 5 * 1000;
```

Comment:
Is this going to be long enough?  Seems like since this is synchronizing across the whole machine, and is protecting syscalls, file I/O, etc., under heavy load in CI might we bump up against this?